src/framework/MOM_error_handler.F90

```diff
@@ -39,14 +39,8 @@ module MOM_error_handler
 
 contains
 
-!> This returns .true. if the current PE is the root PE.
-function is_root_pe()
-  ! This returns .true. if the current PE is the root PE.
-  logical :: is_root_pe
-  is_root_pe = .false.
-  if (mpp_pe() == mpp_root_pe()) is_root_pe = .true.
-  return
-end function is_root_pe
+! is_root_pe returns .true. if the current PE is the root PE.
```

Comment:
Shouldn't this comment be deleted now?

Reply:
This comment was left here deliberately because is_root_pe() is an interface that is available from MOM_error_handler.F90, even though it is a simple pass through from MOM_error_infra.F90 (which will be soon moved into the infrastructure directory).  However, if you think that this is confusing or unhelpful, @adcroft, this comment could be deleted.

  In general, I think that it might be a good idea to include comments describing any pass-through routines within the gateway modules that will be staying in MOM6/src/framework.